src/v0/endpoints/graphql/routes.rs

```diff
@@ -0,0 +1,34 @@
+use super::schema::QueryRoot;
+use crate::db;
+use crate::v0::context::SharedContext;
+use async_graphql::{http::GQLResponse, EmptyMutation, EmptySubscription, QueryBuilder, Schema};
+use std::convert::Infallible;
+use std::sync::Arc;
+use warp::{filters::BoxedFilter, Filter, Rejection, Reply};
+
+pub fn filter(
+    root: BoxedFilter<()>,
+    _context: SharedContext,
+    db_url: &str,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    // load a connection pool for the graphql schema
+    let db_connection_pool =
```

Comment:
This should not be here and will be loaded in main with the db configuration (url).